public/app/plugins/datasource/loki/components/monaco-query-field/monaco-completion-provider/completions.ts

```diff
@@ -14,7 +14,8 @@ export type CompletionType =
   | 'PATTERN'
   | 'PARSER'
   | 'LINE_FILTER'
-  | 'LINE_FORMAT';
+  | 'LINE_FORMAT'
```

Comment:
It seems like we don't use `LINE_FORMAT` anywhere anymore

Reply:
True.